### PR TITLE
Updated benchmark list & targets

### PR DESCRIPTION
**Pull request by petermattson**
Resnet is v1.5
Deep speech delayed
Targets updated

### CODE REVIEW

**Comment by petermattson** [comment]
Fixes #41, fixes #46, fixes #45, fixes #70, fixes #35

File: training_rules.adoc [hunk 1]
```diff
@@ -88,16 +88,23 @@ The benchmark suite consists of the benchmarks shown in the following table.
 |===
 |Area|Problem |Dataset |Quality Target
 
-|Vision |Image classification |ImageNet |74.90% classification
-| |Object detection (light weight) |COCO |TBD
+|Vision |Image classification |ImageNet |75.90% classification
+| |Object detection (light weight) |COCO |21.2% mAP
```
Comment (nvpaulius) [inline training_rules.adoc]: I'd say let's change this to 0.212, to be consistent with mAP notation for the heavy weight detector.

File: training_rules.adoc [hunk 2]
```diff
@@ -88,16 +88,23 @@ The benchmark suite consists of the benchmarks shown in the following table.
...
 | |Object detection (heavy weight) |COCO |0.377 Box min AP and 0.339 Mask min AP
```
Comment (nvpaulius) [inline training_rules.adoc]: Technically Mask R-CNN is "Object detection and instance segmentation"

**Review by nvpaulius — APPROVED** [review]

**Comment by samgd** [comment]
@petermattson Can you clarify the "Deep speech delayed" statement? Does this affect both the open and closed track for the Oct 31 submission deadline or just the closed track? How long actually _is_ the delay (i.e. what is the expected cycle time between submission rounds)?

The only public information about this delay I can find on the forums is in the August meeting slides which states "Delay DeepSpeech2 result submission by 1-2 months to allow for further hyperparameter tuning".

**Comment by petermattson** [comment]
Hi Sam,

It's delayed until the second submission round, which is tentatively
scheduled for January. I'll make the statement clearer.

We enlarged the model (by ~4x) to make it more consistent with industry
practice very late in the v0.5 process, and the folks working on it are
still working on good hyper parameter settings.

Best,

Peter

File: training_rules.adoc (hunk 1 above)
Comment (nvpaulius) [reply training_rules.adoc]: Actually, #72 should be discussed in the submitters working group. I'd like to make sure we have data that the reference reaches target accuracy consistently, and also at larger batch sizes.

**Comment by samgd** [comment]
Hi Peter,

Thanks for the update! We have had an entry provisionally ready for submission to the speech-to-text open track since the first deadline so it's disappointing to hear that the goal post has been moved yet again due to issues with the baseline implementation.

Sam

**Comment by petermattson** [comment]
Samg -- Great to hear you're planning a submission, sorry about the delay. We're actually looking for another owner for the benchmark, any interest?
Paulius -- Backed out the ResNet target per ongoing Submitters WG discussion. Merging the rest.

**Comment by samgd** [comment]
@petermattson: Yes, we're definitely interested - who should we contact?

**Comment by petermattson** [comment]
Hi Sam, please email me at petermattson@google.com. Thanks, Peter

**Pull request merged**